File: src/main/webapp/src/app/models/guest.ts
export class Guest {
  id: number | undefined;
  name: string | null | undefined;
  note: string | null | undefined;

  constructor(name: string | null | undefined, note: string | null | undefined) {
    this.id = undefined;
    this.name = name;
    this.note = note;
  }

}

File: src/main/java/com/seyidov/k8s/entity/Guest.java
package com.seyidov.k8s.entity;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.ToString;

@AllArgsConstructor
@NoArgsConstructor
@Getter
@ToString
@Entity
public class Guest {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    private String name;
    private String note;

}
